app/app.py

```diff
@@ -120,19 +112,19 @@ def authenticate():
           properties:
             username:
               type: string
-              description: The user's SRN or PRN
+              description: The user's SRN, PRN, email, or phone number (validated and trimmed)
               example: PES1UG20CS123
             password:
               type: string
-              description: The user's password
+              description: The user's password (must not be empty)
               example: yourpassword
             profile:
               type: boolean
-              description: Whether to fetch the user's profile
+              description: Whether to fetch the user's profile (strict boolean validation)
               default: false
             fields:
               type: array
-              description: List of profile fields to return. Must be from the predefined set of allowed fields.
+              description: List of profile fields to return (validated against predefined allowed fields)
```

Comment:
Revert these changes. Same for the ones below. We do not need to add these to the API docs.